Weather provider weatherflow

10bias opened the pull request:
Hi everyone,

i added a weather provider for weatherflow (https://weatherflow.com/)

best regards

khassel commented:
Thank you for the contribution. 

An additional PR in the [documentation repo](https://github.com/MichMich/MagicMirror-Documentation) would be helpful, a new section in this [document](https://github.com/MichMich/MagicMirror-Documentation/blob/master/modules/weather.md) for the new provider.

rejas commented on modules/default/weather/providers/weatherflow.js:
```diff
@@ -0,0 +1,86 @@
+
+/* MagicMirror²
+ * Module: Weather
+ * Provider: Dark Sky
```
wrong name :-)

rejas commented:
Also you probably need to do a "npm run lint:prettier" and push the changes so that the linter wont complain once @MichMich approves the workflows

10bias commented:
Thanks for the hints. I fixed the copy and paste mistake in the comment and run the lint. I also added a documentation for the provider und created a pr: https://github.com/MichMich/MagicMirror-Documentation/pull/109

10bias commented:
I have just add a fix with the sunrise.

rejas commented on modules/default/weather/providers/weatherflow.js:
```diff
@@ -0,0 +1,81 @@
+		stationid: ""
+	},
+
+	units: {
```
those seem unused?

10bias commented:
@rejas oh yes, i have added a functionality to switch between the units. I hope the imperial units a correct. Some researches didn't deliverd a clear result. If there is something wrong, please let me know.

rejas commented:
Thx for the info. Cant tell if yomethings wrong, got no api keys :-) Since this a fresh provider, I dont mind merging it and see if it works "in the wild" :-)

MichMich commented:
Thanks for your contribution! My apologies for my slow response. Life got in the way.
